server/play-server/src/main/scala/sttp/tapir/server/play/PlayServerOptions.scala

```diff
@@ -36,7 +38,7 @@ object PlayServerOptions {
           SingletonTemporaryFileCreator,
           defaultDeleteFile(_),
           DefaultActionBuilder.apply(PlayBodyParsers.apply().anyContent),
```

Comment:
**question:** shouldn't we pass `ParserConfiguration` to `apply()` here as well?

Reply:
I'm not sure abut that but I will add to have the same PlayBodyParser.